docs/src/main/asciidoc/feature-management.adoc

```diff
@@ -0,0 +1,276 @@
+== Feature Management
+
+Feature Management provide a way for Spring Boot applications dynamically access content. Feature Management has a verity of functions, such as Feature Flags that can enable or disable content, Feature Flags for targeting when content is shown, and Feature Gates for dynamically enabling endpoints.
+
+By default, loading of Feature Flags from Azure App Configuration is disabled. And is enabled through configuration:
+
+[source,properties,indent=0]
+----
+spring.cloud.azure.appconfiguration.stores[0].feature-flags.enabled= true
+----
+
+With Feature Flags enabled they will be loaded into the Spring Configuration system with the prefix `feature-management`. Feature Flags can also be registered in the local configuration file, see <<Feature Flag Declaration>>. The easiest was to use Feature Management is by using the `azure-spring-cloud-feature-management` and `azure-spring-cloud-feature-management-web` libraries. The difference between the two libraries is that `azure-spring-cloud-feature-management-web` takes a dependency on the `spring-web` and `spring-webmvc` libraries to add additional features, such as Feature Gates.
+
+=== Feature Management Basics
+
+==== Feature Flags
+
+Feature flags are composed of two parts, a name and a list of feature-filters that are used to turn the feature on. Feature Flags can either have a boolean state of on/off, or they can have a list of Feature Filters. Feature Flags evaluate Feature Filters until one returns true, if no Feature Filter returns true, then the Feature Flag returns false.
+
+==== Feature Filters
+
+Feature filters define a scenario for when a feature should be enabled. Feature Filters are evaluated synchronously.
```

Comment:
Would mention here that we support 3 types of feature filters, the details of them can be in their own sections.

Reply:
Updated, but it is 4 not 3.